src/Cards/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import { CardsContainer, Score } from "./styled";
+
+import CurrentCard from "./CurrentCard";
+import Finish from "./Finish";
+import { triviaQuestions } from "./triviaQuestions";
+
+const total_questions = triviaQuestions.length;
+const currentRoundQuestions = [];
+
+const Cards = () => {
+  const [score, setScore] = useState(0);
+  const [currentQuestion, setCurrentQuestion] = useState(undefined);
+  const [isSubmit, setIsSubmit] = useState(false);
```

Comment:
I would rename this to `hasSubmitted` to convey that the user has already submitted